applications/RANSModellingApplication/custom_processes/auxiliary_processes/rans_wall_distance_calculation_process.h

```diff
@@ -0,0 +1,320 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya (https://github.com/sunethwarna)
+//
+
+#if !defined(KRATOS_RANS_WALL_DISTANCE_CALCULATION_PROCESS_H_INCLUDED)
+#define KRATOS_RANS_WALL_DISTANCE_CALCULATION_PROCESS_H_INCLUDED
+
+// System includes
+#include <string>
+
+// External includes
+
+// Project includes
+#include "containers/model.h"
+#include "custom_utilities/rans_variable_utils.h"
+#include "includes/cfd_variables.h"
+#include "includes/checks.h"
+#include "includes/define.h"
+#include "factories/linear_solver_factory.h"
+#include "includes/model_part.h"
+#include "processes/process.h"
+#include "processes/variational_distance_calculation_process.h"
+#include "rans_modelling_application_variables.h"
+
+namespace Kratos
+{
+///@addtogroup RANSModellingApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+template <class TSparseSpace, class TDenseSpace, class TLinearSolver>
```

Comment:
why the templates?

Reply:
Because VariationalDistanceCalculation need the templates. This is just a wrapper around it to be used in json files